setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
Well, been using Python professionally for ~ 5 years now, which doesn't make me a real expert either :). Came across your project when paul-wolf mentioned it on python-list and think it's quite a nice idea. Keep going!